app/app.js

```diff
@@ -1,9 +1,10 @@
-import Ember from 'ember';
-import Resolver from './resolver';
+import Application from '@ember/application';
 import loadInitializers from 'ember-load-initializers';
+
+import Resolver from './resolver';
```

Comment:
my convention is usually to import external packages first before importing local files. since `ember-load-initializers` is external and `./resolver` is a local file the resolver is imported afterwards